package.json

```diff
@@ -52,16 +52,16 @@
     "@sentry/react": "^7.8.1",
     "@sentry/tracing": "^7.8.1",
     "@truffle/hdwallet-provider": "^2.0.14",
-    "@web3-onboard/coinbase": "^2.1.2",
-    "@web3-onboard/core": "^2.8.2",
-    "@web3-onboard/fortmatic": "^2.0.13",
-    "@web3-onboard/injected-wallets": "^2.2.1",
-    "@web3-onboard/keystone": "^2.3.0",
-    "@web3-onboard/ledger": "^2.3.0",
-    "@web3-onboard/portis": "^2.1.2",
-    "@web3-onboard/torus": "^2.1.2",
-    "@web3-onboard/trezor": "^2.3.0",
-    "@web3-onboard/walletconnect": "^2.1.2",
+    "@web3-onboard/coinbase": "^2.1.3-alpha.1",
+    "@web3-onboard/core": "2.8.5-alpha.2",
+    "@web3-onboard/fortmatic": "^2.0.14-alpha.1",
+    "@web3-onboard/injected-wallets": "^2.2.4-alpha.2",
+    "@web3-onboard/keystone": "^2.3.1-alpha.1",
+    "@web3-onboard/ledger": "^2.3.1-alpha.1",
+    "@web3-onboard/portis": "^2.1.3-alpha.1",
+    "@web3-onboard/torus": "^2.1.3-alpha.1",
+    "@web3-onboard/trezor": "^2.3.1-alpha.1",
+    "@web3-onboard/walletconnect": "^2.1.3-alpha.1",
```

Comment:
An official release will be available [soon](https://github.com/blocknative/web3-onboard/issues/1287). Although "alpha", it [only updates a peer dependency](https://github.com/blocknative/web3-onboard/pull/1290).

Reply:
Should we wait with the update until the official release is published and its not alpha anymore?

Reply:
As seen in [their PR](https://github.com/blocknative/web3-onboard/pull/1290), they are only updating `joi` to the same version across packages. If we prefer to not use the alpha versions then we need to have static dependency versions instead.

Reply:
I would prefer we don't upgrade to pre-releases unless its a critical fix that is needed. Which command did you run to upgrade the versions? I tried `yarn upgrade` but it doesn't change anything so maybe the caret already excludes postfix versions.

Reply:
The inclusion of the caret automatically upgrades to all minor/patch versions and the current version is broken. I referenced their PR for these packages.